docs/basic-setup.md

```diff
@@ -48,4 +48,4 @@ Afterwards, the simple Hello World program is executed with:
 
 ```bash
 ./som core-lib/Hello.ns
-```
+```
```

Comment:
Strange change?

Reply:
Yes, the change should be removed but I don't know how. I checked out from dev and it is still there.